apps/helpers.py

```diff
@@ -245,6 +246,26 @@ def create_instance_from_form(form, project, app_slug, app_id=None):
     """
     from .tasks import deploy_resource
 
+    new_app = app_id is None
+
+    logger.debug(f"Creating or updating a user app via UI form for app_id={app_id}, new_app={new_app}")
+
+    # Do not deploy resource for edits that do not require a k8s re-deployment
+    do_deploy = False
+
+    if new_app:
+        do_deploy = True
+    else:
+        # Only re-deploy existing apps if one of the following fields was changed:
+        redeployment_fields = ["subdomain", "volume", "path", "flavor", "port", "image"]
```

Comment:
I feel like this is worthy of being in the `settings.py`

Reply:
This list should be the same in all environments and modifying it essentially would require starting in development and rebuilding and testing it anyway in all environments so I do not really see what value moving this to settings would give us.